clang/test/CIR/CodeGen/ifstmt.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -fclangir-enable -emit-cir %s -o - | FileCheck %s
```

Comment:
NIT:

Could you move the tests in `clang/tests/CIR/Codegen/spelling-locations.cpp` into this file?

They both test if statement location issues.